custom/icds/templates/icds/ccz_hosting.html

```diff
@@ -34,6 +34,11 @@ <h3>
         <div class="row">
             <div class="col-sm-1"></div>
             <div class="col-sm-9">
+                {% if icds_env %}
```

Comment:
thats a fair point. Just added this in case this is used by any other project/domain/environment like staging, though i could not get this to work on staging because icds-ucr is not configured so the testing might happen on india.